autoscaling_group.go

```diff
@@ -50,9 +50,9 @@ type AutoscalingGroupListParams struct {
 
 type AutoscalingGroupUpdateAttributeParams struct {
 	Name        string `json:"name,omitempty"`
-	Min         int    `json:"min,omitempty"`
-	Max         int    `json:"max,omitempty"`
-	Current     int    `json:"current,omitempty"`
+	Min         *int   `json:"min,omitempty"`
+	Max         *int   `json:"max,omitempty"`
+	Current     *int   `json:"current,omitempty"`
```

Comment:
Can you do this for other entities in a followup PR? Relatedly, should all fields be pointers?